controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -109,3 +472,17 @@ func (r *StorageClassClaimReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&ocsv1alpha1.StorageClassClaim{}).
 		Complete(r)
 }
+
+func (r *StorageClassClaimReconciler) get(obj client.Object) error {
+	key := client.ObjectKeyFromObject(obj)
+	return r.Client.Get(r.ctx, key, obj)
+}
+
+func contains(slice []string, s string) bool {
+	for _, item := range slice {
+		if item == s {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
This function is duplicated in multiple parts of codebase. Just reuse it.

Reply:
+1